src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int numberOfBalls = 100;
```

Comment:
make it constant. [See here about constants and naming convention ](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)